classes/manager.php

```diff
@@ -1216,15 +1216,9 @@ public function validate_component_grade(int $componentgradeid, string $sourcety
     public function get_all_course_activities(int $courseid): array {
         $activities = [];
         foreach (self::allowed_activities() as $modname) {
-            // Skip if the activity is not installed.
-            if (!array_key_exists($modname, core_component::get_plugin_list('mod'))) {
-                continue;
-            }
```

Comment:
did this check move elsewhere?

Reply:
The `self::allowed_activities()` function above has already cover this check
https://github.com/ucl-isd/moodle-local_sitsgradepush/blob/main/classes/manager.php#L1541

Reply:
> #70 was dealing with this in a check in classes/output/renderer.php but I don't see it in this consolidated branch
This `can_map_to_mab()` function call above has covered that checking.
https://github.com/ucl-isd/moodle-local_sitsgradepush/blob/main/classes/assessment/assessment.php#L101